### ISSUE TITLE
Indicators/Disaggs

### ISSUE THREAD

**Issue by achafetz** [noahbartlett/Me_One_Last_Kiss#7]
Are all indicators needed? Highlighted areas of overlap but there may also be indicators that aren't used.

https://github.com/noahbartlett/Me_One_Last_Kiss/blob/365e5ad71241bb4dcb4966ea46afb15973d3522a/R%20Files/02_FY18Q2Tableau.R#L45-L57

![image](https://user-images.githubusercontent.com/8933069/42052002-98a7b81c-7adb-11e8-9aef-093eb66d7c63.png)

**Comment by achafetz** [comment]
And follow up question is whether all these 158 disaggs are used in the Tableau report. could be an area to pair down

indicator | standardizeddisaggregate
-- | --
GEND_GBV | Age/Sex
GEND_GBV | Age/Sex/ViolenceType
GEND_GBV | PEP
GEND_GBV | Total Numerator
GEND_GBV | ViolenceServiceType
HRH_PRE | Grad Cadre
HRH_PRE | Total Numerator
HTS_SELF | Age/Sex/HIVSelfTest
HTS_SELF | HIVSelfTest
HTS_SELF | HIVSelfTestUser
HTS_SELF | KeyPop/HIVSelfTest
HTS_SELF | Total Numerator
HTS_TST | Age Aggregated/Sex
HTS_TST | Age Aggregated/Sex/Result
HTS_TST | Age/Sex/Result
HTS_TST | AgeAboveTen/Sex
HTS_TST | AgeAboveTen/Sex/Positive
HTS_TST | AgeLessThanTen
HTS_TST | AgeLessThanTen/Positive
HTS_TST | CommunityDeliveryPoint
HTS_TST | FacilityDeliveryPoint
HTS_TST | KeyPop/Result
HTS_TST | Modality/Age Aggregated/Sex/Result
HTS_TST | Modality/Age/Sex/Result
HTS_TST | Modality/MostCompleteAgeDisagg
HTS_TST | MostCompleteAgeDisagg
HTS_TST | Positive
HTS_TST | Results
HTS_TST | ServiceDeliveryPoint
HTS_TST | ServiceDeliveryPoint/Result
HTS_TST | Total Numerator
HTS_TST_NEG | Age Aggregated/Sex/Result
HTS_TST_NEG | Age/Sex/Result
HTS_TST_NEG | KeyPop/Result
HTS_TST_NEG | Modality/Age Aggregated/Sex/Result
HTS_TST_NEG | Modality/Age/Sex/Result
HTS_TST_NEG | Modality/MostCompleteAgeDisagg
HTS_TST_NEG | Results
HTS_TST_NEG | ServiceDeliveryPoint/Result
HTS_TST_NEG | Total Numerator
HTS_TST_POS | Age Aggregated/Sex/Result
HTS_TST_POS | Age/Sex/Result
HTS_TST_POS | AgeAboveTen/Sex/Positive
HTS_TST_POS | AgeLessThanTen/Positive
HTS_TST_POS | KeyPop/Result
HTS_TST_POS | Modality/Age Aggregated/Sex/Result
HTS_TST_POS | Modality/Age/Sex/Result
HTS_TST_POS | Modality/MostCompleteAgeDisagg
HTS_TST_POS | MostCompleteAgeDisagg
HTS_TST_POS | Positive
HTS_TST_POS | ServiceDeliveryPoint/Result
HTS_TST_POS | Total Numerator
KP_MAT | Sex
KP_MAT | Total Numerator
KP_PREV | KeyPop
KP_PREV | KeyPop/Status
KP_PREV | Total Denominator
KP_PREV | Total Numerator
OVC_SERV | Age/Sex
OVC_SERV | Age/Sex/Service
OVC_SERV | ProgramStatus
OVC_SERV | Total Numerator
OVC_SERV | TransferExit
OVC_SERV_OVER_18 | Total Numerator
OVC_SERV_UNDER_18 | Total Numerator
PMTCT_ART | MaternalRegimenType
PMTCT_ART | NewExistingArt/Sex/HIVStatus
PMTCT_ART | Total Denominator
PMTCT_ART | Total Numerator
PMTCT_EID | Age
PMTCT_EID | Age/HIVStatus
PMTCT_EID | InfantTest
PMTCT_EID | Total Numerator
PMTCT_EID_Less_Equal_Two_Months | Total Numerator
PMTCT_EID_POS | Total Numerator
PMTCT_EID_Two_Twelve_Months | Total Numerator
PMTCT_STAT | Age/Sex
PMTCT_STAT | Age/Sex/KnownNewResult
PMTCT_STAT | Known/New
PMTCT_STAT | Known/New/Age
PMTCT_STAT | Total Denominator
PMTCT_STAT | Total Numerator
PMTCT_STAT_KnownatEntry_POSITIVE | Total Numerator
PMTCT_STAT_NewlyIdentified_Negative | Total Numerator
PMTCT_STAT_NewlyIdentified_POSITIVE | Total Numerator
PMTCT_STAT_POS | Age/Sex/KnownNewResult
PMTCT_STAT_POS | Known/New
PMTCT_STAT_POS | Known/New/Age
PMTCT_STAT_POS | Total Numerator
PP_PREV | Age Aggregated/Sex
PP_PREV | Age/Sex
PP_PREV | Status
PP_PREV | Total Denominator
PP_PREV | Total Numerator
PrEP_NEW | Age/Sex
PrEP_NEW | KeyPopAbr
PrEP_NEW | Total Numerator
TB_ART | Age
TB_ART | Age Aggregated/Sex
TB_ART | Age Aggregated/Sex/HIVStatus
TB_ART | Age/Sex
TB_ART | Age/Sex/HIVStatus
TB_ART | NewExistingArt/HIVStatus
TB_ART | Sex
TB_ART | Total Denominator
TB_ART | Total Numerator
TB_STAT | Age
TB_STAT | Age Aggregated/Sex
TB_STAT | Age Aggregated/Sex/KnownNewPosNeg
TB_STAT | Results
TB_STAT | Sex
TB_STAT | Total Denominator
TB_STAT | Total Numerator
TX_CURR | Age Aggregated/Sex/HIVStatus
TX_CURR | Age/Sex/HIVStatus
TX_CURR | MostCompleteAgeDisagg
TX_CURR | Total Numerator
TX_NEW | Age Aggregated/Sex/HIVStatus
TX_NEW | Age/Sex/HIVStatus
TX_NEW | KeyPop/HIVStatus
TX_NEW | MostCompleteAgeDisagg
TX_NEW | PregnantOrBreastfeeding/HIVStatus
TX_NEW | TB Diagnosis
TX_NEW | TB Diagnosis/HIVStatus
TX_NEW | Total Numerator
TX_PVLS | Age Aggregated/Sex
TX_PVLS | Age Aggregated/Sex/Indication/HIVStatus
TX_PVLS | Age/Sex/Indication/HIVStatus
TX_PVLS | PregnantOrBreastfeeding/Indication/HIVStatus
TX_PVLS | RoutineTargeted
TX_PVLS | RoutineTargeted/HIVStatus
TX_PVLS | Total Denominator
TX_PVLS | Total Numerator
TX_RET | 12mo
TX_RET | 24mo
TX_RET | 36mo
TX_RET | Age Aggregated/Sex
TX_RET | Age/Sex
TX_RET | PregnantOrBreastfeeding/HIVStatus
TX_RET | Total Denominator
TX_RET | Total Numerator
TX_TB | Age Aggregated/Sex
TX_TB | Age Aggregated/Sex/HIVStatus
TX_TB | HIVStatus
TX_TB | NewExistingArt/HIVStatus
TX_TB | PositiveScreen
TX_TB | Specimen Sent
TX_TB | Specimen Sent/HIVStatus
TX_TB | TB Test Type/HIVStatus
TX_TB | TBScreen/NewExistingART/HIVStatus
TX_TB | Total Denominator
TX_TB | Total Numerator
VMMC_CIRC | Age/Sex
VMMC_CIRC | HIVStatus/Sex
VMMC_CIRC | TechFollowUp/Sex
VMMC_CIRC | TechFollowUp>14days/Sex
VMMC_CIRC | Technique/Sex
VMMC_CIRC | Total Numerator

**Comment by noahbartlett** [comment]
No doubt the disaggregates can be greatly reduced. I will go over them and figure out what we aren't using. 

The indicators, I am less sure of. I definitely want to keep HTS_POS and PMTCT_POS since those are things people look for. OVC_UNDER_18 is important too. ( I know they can be recreated using disaggregates). I need to noodle on this. Especially since:

HTS_TST_POS (Total num) / HTS_TST_POS (Total num) + HTS_TST_NEG (Total num) !=
HTS_TST (positive) / HTS_TST (positive) + HTS_TST (negative)

**Comment by achafetz** [comment]
Yeah, have a look and let me know what we need and we can really reduce some of the file size I think/hope.

As for HTS, there are a handful of countries where HTS_TST_POS + HTS_TST_NEG <> HTS_TST (total numerator) +/- 5% since the start of FY17.

The below chart shows countries where their disaggregate percent completeness is greater than +/-5% of total numerator. Malawi seems to be the only one with major issues (due to MWI MoH reporting structure)
![image](https://user-images.githubusercontent.com/8933069/42093439-4aa8186a-7b7a-11e8-9d0f-d4c5900a1a3b.png)
